study-builder/studies/pancan/prequal.conf

```diff
@@ -143,83 +143,15 @@
           "blockType": "QUESTION",
           "shownExpr": null
         },
-        {
```

Comment:
apparently advanced-breast/prostate question is needed  only if self_only option is selected.
hence removed body template prompts for self , child and added self prompt to question.

Reply:
Ah nice. thanks for catching that! No wonder I didn't see mention of a "child" variant of the text in the spreadsheet...

Reply:
Wait... when they select both "self" and "child" diagnosed options, the behavior is that we treat it as if it's just "self". That's why I had put in checks like `is_self` and `is_child_only`. If they selected both "self" and "child", I think we should still show advanced breast/prostate questions, since it's technically a "self" case?

Reply:
If so: They can never enter child stuff right, because its redirected to existing study

Reply:
I think so. In the "self only" and "self+child" cases, we're only displaying self questions, so don't think they'll answer any child questions.

Reply:
we can change it later if needed. flow chart has  a 3rd diff flow if self + child are chosen. that doesn't have any mention of study-redirects

Reply:
Here either way advanced question is only asked for "self" , we should be fine here.